tests/operations/PatchOperationListTest.php

```diff
@@ -0,0 +1,251 @@
+<?php declare(strict_types=1);
+
+namespace blancks\JsonPatchTest\operations;
+
+use blancks\JsonPatch\exceptions\FastJsonPatchExceptionTrait;
+use blancks\JsonPatch\exceptions\InvalidPatchException;
+use blancks\JsonPatch\exceptions\InvalidPatchOperationException;
+use blancks\JsonPatch\json\accessors\ArrayAccessor;
+use blancks\JsonPatch\json\accessors\ArrayAccessorAwareTrait;
+use blancks\JsonPatch\json\accessors\ObjectAccessor;
+use blancks\JsonPatch\json\accessors\ObjectAccessorAwareTrait;
+use blancks\JsonPatch\json\accessors\ValueAccessorAwareTrait;
+use blancks\JsonPatch\json\handlers\BasicJsonHandler;
+use blancks\JsonPatch\json\handlers\JsonHandlerInterface;
+use blancks\JsonPatch\operations\Add;
+use blancks\JsonPatch\operations\Copy;
+use blancks\JsonPatch\operations\Move;
+use blancks\JsonPatch\operations\PatchOperation;
+use blancks\JsonPatch\operations\PatchOperationList;
+use blancks\JsonPatch\operations\Remove;
+use blancks\JsonPatch\operations\Replace;
+use blancks\JsonPatch\operations\Test;
+use blancks\JsonPatchTest\JsonPatchCompliance;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\Attributes\UsesClass;
+use Throwable;
+
+#[CoversClass(PatchOperationList::class)]
+#[UsesClass(Add::class)]
+#[UsesClass(Copy::class)]
+#[UsesClass(Move::class)]
+#[UsesClass(Remove::class)]
+#[UsesClass(Replace::class)]
+#[UsesClass(Test::class)]
+#[UsesClass(PatchOperation::class)]
+#[UsesClass(FastJsonPatchExceptionTrait::class)]
+#[UsesClass(InvalidPatchException::class)]
+#[UsesClass(InvalidPatchOperationException::class)]
+#[UsesClass(ArrayAccessor::class)]
+#[UsesClass(ArrayAccessorAwareTrait::class)]
+#[UsesClass(ObjectAccessor::class)]
+#[UsesClass(ObjectAccessorAwareTrait::class)]
+#[UsesClass(ValueAccessorAwareTrait::class)]
```

Comment:
I'm not familiar with running phpunit with the `requireCoverageMetadata` / `beStrictAboutCoverageMetadata` options configured. 

I *think* I've done this correctly (certainly got the coverage build to go green, after a few attempts) although I found it a bit surprising - it feels like the test now has a dependency on implementation details that it shouldn't be aware of. Perhaps that's just the necessary trade-off but let me know if there's something different I should have done.